### DISCUSSION TITLE
Sharing Teleport Locations

### DISCUSSION THREAD

**Discussion by TexasToast03** [YimMenu/HorseMenu#193] [Q&A]
Anyway for me to be able to share custom location teleports here? I was bored and went to almost every location and made some hidden/discrete spawn points for cities. Idk if this is even possible to share so I was just checking.

**Comment by tuyilmaz** [comment]
Go to %appdata%/HorseMenu and share the telelocations.json file

**Reply by TexasToast03** [reply]
Alright, thanks. Im finishing up some locations. Ill try this out tomarrow which is technically today for me

**Reply by TexasToast03** [reply]
> Go to %appdata%/HorseMenu and share the telelocations.json file

Would i post it here or where?

**Reply by tuyilmaz** [reply]
Uhmm... yeah you can post it here.

**Reply by TexasToast03** [reply]
Ok

**Comment by TexasToast03** [comment]
[telelocations.json](https://github.com/user-attachments/files/16513029/telelocations.json)
